popSimManu.tex

```diff
@@ -230,9 +230,9 @@ \section*{Introduction}
 These available models and parameters are meant to represent the field's current understanding,
 and we intend for this resource to evolve as new results become available, and relevant
 existing models are added to \stdpopsim by the community.
-Through either a command line interface or a simple Python API, users can specify which
-organism, genetic map, chromosome, and demographic history they are interested in simulating, and the
-simulation output from their chosen model is returned.
+We have implemented both a command line interface and a simple Python API
+that can be used to simulate genomic data
+from a choice of organism, genetic map, chromosome, and demographic history.
```

Comment:
"returning simulation output" seemed redundant - why would you simulate if not to get output?